sqlite3/test/database_test.dart

```diff
@@ -82,6 +82,51 @@ void main() {
               'INSERT INTO foo VALUES (?); INSERT INTO foo VALUES (?);', [123]),
           throwsArgumentError);
     });
+
+    test('inner join with toTableColumnMap and computed column', () {
```

Comment:
I think we would have to skip this test if `sqlite3_column_table_name` is not available.

You can do something like

```dart
final hasColumnMeta = open.openSqlite().providesSymbol('sqlite3_column_table_name');

test('inner join with toTableColumnMap and computed column', () {
  // ...
}, skip: hasColumnMeta ? null : 'sqlite3 was compiled without column metadata');
```